src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.StockBalance;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String COLUMN_NAMES = "fruit,quantity";
+    private static final String DELIMITER = ",";
+
+    @Override
+    public String get(Map<String, Integer> balance) {
```

Comment:
```suggestion
    public String getReport() {
```
Make Map class field and inject using constructor.